tests/node/tests/operations.mjs

```diff
@@ -1076,5 +1076,296 @@ ExifImageHeight: 57`);
     }),
 
 
+    it("performs MAGIC", async () => {
+        const input = "WUagwsiae6mP8gNtCCLUFpCpCB26RmBDoDD8PacdAmzAzBVjkK2QstFXaKhpC6iUS7RHqXrJtFisoRSgoJ4whjm1arm864qaNq4RcfUmLHrcsAaZc5TXCYifNdgS83gDeejGX46gaiMyuBV6EskHt1scgJ88x2tNSotQDwbGY1mmCob2ARGFvCKYNqiN9ipMq1ZU1mgkdbNuGcb76aRtYWhCGUc8g93UJudhb8htsheZnwTpgqhx83SVJSZXMXUjJT2zmpC7uXWtumqokbdSi88YtkWDAc1Toouh2oH4D4ddmNKJWUDpMwmngUmK14xwmomccPQE9hM172APnSqwxdKQ172RkcAsysnmj5gGtRmVNNh2s359wr6mS2QRP";
+        const output = [
```

Comment:
Magic output is likely to change on a reasonably regular basis as we tweak settings. I'd rather this test wasn't quite so specific otherwise we'll have to change it continuously!

Reply:
Yep, fair. I'll spend some time thinking about which parts of this are important to test for.